src/components/App.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
[Optional]

We can use a stateless function component for this milestone.

But don't forget to remove this when the component becomes stateful.

Reply:
I thought that it would be stateful in next milestone or later, so I made just this one component a stateful component.

Reply:
No problem, you are equally right! :smile: